doc/examples/denoise_patch2self.py

```diff
@@ -68,10 +68,18 @@
 hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
 data, affine = load_nifti(hardi_fname)
 bvals = np.loadtxt(hardi_bval_fname)
-denoised_arr = patch2self(data, bvals, shift_intensity=True,
-                          clip_negative_vals=False)
+denoised_arr = patch2self(data, bvals, model='ols', shift_intensity=True,
+                          clip_negative_vals=False, b0_denoising=50)
```

Comment:
```suggestion
                          clip_negative_vals=False, b0_threshold=50)
```
More here: I believe that `b0_denoising` is a boolean.

Reply:
Or possibly
```suggestion
                          clip_negative_vals=False, b0_denoising=True)
```

But both are the default values, so it's not clear why you are setting either of them.

Reply:
My bad! Good catch -- fixing this.